crates/relayer/src/chain/cosmos/gas.rs

```diff
@@ -43,7 +43,19 @@ pub async fn dynamic_gas_price(
     rpc_address: &Url,
 ) -> GasPrice {
     if config.dynamic_gas_price.enabled {
-        let dynamic_gas_price = query_eip_base_fee(rpc_address)
+        let url = if chain_id.name().contains("osmosis") {
+            format!(
+                "{}abci_query?path=\"/osmosis.txfees.v1beta1.Query/GetEipBaseFee\"",
```

Comment:
Was asking because I find it awkward to have hardcoded strings like this and would be nice if we can avoid it. The `fee_market_module` would be something like `EIP-1559` (in case other chains use the same) or `FeeMarket` but yes the fact that the query response is different in the first place is another annoyance and I wish they could be aligned. They are not that much different actually and maybe it wouldn't be hard to ask/ have a consistent response.

But we can wait and see if there is convergence on the skip's module or we start seeing more dynamic fee impls.

Reply:
Makes sense! But just to recap my point was that there is no benefit to making this configurable  because we need bespoke code for handling the potentially different responses format and we cannot just try and deserialize the response o to one format or the other as the various responses might all be proto if compatible and deserialize into each other but with wrong results.

> But we can wait and see if there is convergence on the skip's module or we start seeing more dynamic fee impls.

Agreed, I think we should indeed wait until we see how many different impls we end up with. Until then I suggest we hide this from users and only special case osmosis for now. After we'll see if we need a more generic way of configuring this or if everybody converges on Skip's module.